contracts/strategies/BaseStrategy.sol

```diff
@@ -132,11 +132,11 @@ abstract contract BaseStrategy is
             totalTokensStaked += _amount;
 
             // Calls our internal deposit function
-            (uint256 amountUsed, uint256 amountStaked) = _deposit(_amount);
+            uint256 amountUsed = _deposit(_amount);
```

Comment:
can we remove the `totalTokensStaked += _amount;` and just overwrite `_amount` with the return value of `_deposit` so to avoid having to remove `_amount` from here `totalTokensStaked = totalTokensStaked - _amount + amountUsed;` ?
we can do
```
_amount = _deposit(_amount);
if (_amount != 0) {
    totalTokensStaked = totalTokensStaked +_amount;
 }
``` 
without line 132